ddtrace/internal/packages.py

```diff
@@ -142,11 +142,11 @@ def _root_module(path: Path) -> str:
             pass
 
     # Bazel runfiles support: we assume that these paths look like
-    # /some/path.runfiles/.../site-packages/<root_module>/...
-    if any(p.suffix == ".runfiles" for p in path.parents):
```

Comment:
@P403n1x87 Do you remember why we had this suffix check? We're looping over `path.parents` anyways, and it's checking for `site-package` plus comparing the name to the third-party.txt